greenlight/tests/functional/toolbar/test_back_forward.py

```diff
@@ -12,6 +12,14 @@ class TestBackForward(FirefoxTestCase):
     def setUp(self):
         FirefoxTestCase.setUp(self)
 
+        with self.marionette.using_context("content"):
+            self.marionette.navigate("about:blank")
+
+        self.marionette.execute_script("""
+            let count = gBrowser.sessionHistory.count;
+            gBrowser.sessionHistory.PurgeHistory(count);
+        """)
+
```

Comment:
This test fails the assertion I added. It relies on having a certain amount of history in the tab so the back button ends up disabled later on. It's only passing now because it runs in a leaked tab from an earlier test.